externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/schema/SchemaHelper.scala

```diff
@@ -39,10 +39,13 @@ object SchemaHelper {
     case _: DecimalType => TTypeId.DECIMAL_TYPE
     case DateType => TTypeId.DATE_TYPE
     case TimestampType => TTypeId.TIMESTAMP_TYPE
+    case tntz if tntz.simpleString.equals("timestamp_ntz") => TTypeId.TIMESTAMP_TYPE
```

Comment:
Do we need also handle this in `toJavaSQLType`

Reply:
getDecimalDigits/getDecimalDigits etc

Reply:
Is TimestampNTZType's hive thrift type TTypeId.TIMESTAMP_TYPE and is jdbc type TIMESTAMP ？

Reply:
Ideally，you are right. but there are some historical reasons. at the very beginning，hive does not support timestamp with local timezone but spark support，and they both have only one and be treated as same. now，they both have 2 types supported and the problem caused by old mapping stands out. I guess we shall learn how spark deal with this mess.

Reply:
The implication is that we need to change the previous Timestamp type to TIMESTAMPLOCALTZ_TYPE ？
```scala
case TimestampType => TTypeId.TIMESTAMPLOCALTZ_TYPE
```

Reply:
> The implication is that we need to change the previous Timestamp type to TIMESTAMPLOCALTZ_TYPE ？
> 
> ```scala
> case TimestampType => TTypeId.TIMESTAMPLOCALTZ_TYPE
> ```

But it seems that our current hive version (2.3.9) does not support it yet